asv_bench/benchmarks/algorithms.py

```diff
@@ -183,4 +183,4 @@ def time_argsort(self, N):
         self.array.argsort()
 
 
-from .pandas_vb_common import setup  # noqa: F401 isort:skip
+from .pandas_vb_common import setup
```

Comment:
These can't be deleted and will still cause errors

Reply:
Okay. Will split and fix. @WillAyd